modules/corelib/const.lua

```
@@ -312,7 +322,18 @@ KeyCodeDescs                 = {
+    [KeyBrowserBack] = 'BrowserBack',
+    [KeyBrowserSearch] = 'BrowserSearch',
+    [KeyBrowserHome] = 'BrowserHome',
+    [KeyVolumeMute] = 'Mute',
+    [KeyVolumeDown] = 'VolumeDown',
+    [KeyVolumeUp] = 'VolumeUp',
+    [KeyMediaNext] = 'MediaNext',
+    [KeyMediaPrev] = 'MediaPrev',
+    [KeyMediaPlayPause] = 'PlayPause',
+    [KeyLaunchMediaSelect] = 'MediaLaunch'
+}
 }
```

Luacheck (1.2.0)

[error] expected statement near '}'

(E011)